vocab_files/observable_property_concepts/damage-area.ttl

```diff
@@ -0,0 +1,19 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schema: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://linked.data.gov.au/def/nrm/a1f5f46b-d6fe-4a1f-8fa8-01b113a2d69f>
+    a skos:Concept ;
+    dcterms:source "Cox B, Peacock D, McCallum K, O’Neill S, Sparrow B. (2023) Herbivory and Physical Damage Module. In ‘Ecological Field Monitoring Protocols Manual using the Ecological Monitoring System Australia’. (Eds S O’Neill, K Irvine, A Tokmakoff, B Sparrow). TERN, Adelaide." ;
+    rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:altLabel
+        "area of damage" ,
+        "damage size" ;
+    skos:closeMatch <https://linked.data.gov.au/def/nrm/f8d5c98b-ecee-5c56-9b95-a57280faec60> ;
```

Comment:
Are they close match? One is area, float measurements. Another one is damage type, selected from a drop-down list.

Reply:
@junrongYu - You can remove the closeMatch for this concept here.